demo/Corona/build.settings

```diff
@@ -9,14 +9,20 @@ settings =
 	orientation =
 	{
 		default = "portrait",
-		supported = { "portrait" }
+		supported = { "portrait","landscapeLeft", "landscapeRight" }
```

Comment:
Got to try this later with Android to see if it doesn't break anything. Shouldn't though 'cause most games are landscape.

Reply:
Yes. Also keep in mind that right now the appToken and the placements are specific for iOS. In addition, we have to implement the change for showing the banner with a position in Android.

Reply:
Yep. I have it in mind. We should maybe later try to separate the tokens as well.